src/main/java/catan/domain/model/game/Resources.java

```diff
@@ -75,6 +76,43 @@ public void setStone(int stone) {
         this.stone = stone;
     }
 
+    public Integer takeResourceQuantity(HexType resource) {
```

Comment:
take is not the best verb)) it is better to leave this name wihout werbs, since it returns only value without any logic, so that you will call this method gameuser.getResources().quantityOf(WOOD);